db/schema.rb

```diff
@@ -154,4 +187,6 @@
     t.datetime "updated_at", null: false
   end
 
+  add_foreign_key "taxonomy_todos", "content_items"
+  add_foreign_key "taxonomy_todos", "taxonomy_projects"
```

Comment:
I think you have some extra tables in your database from other branches. You might need to remove these changes from the branch and amend the commit.